### ISSUE TITLE
No 1.17 Version?

### ISSUE THREAD

**Issue by Aceplante** [PieKing1215/InvMove#50]
I have a 1.17.1 modpack, and both the 1.16.5 and 1.18.1 versions crash, with the 1.16.5 expecting a non-existent version of cloth config? Which should I use?

![issue](https://github.com/PieKing1215/InvMove/assets/101857505/31a6b575-2a51-43f3-b72b-71b7069c772a)

**Comment by PieKing1215** [comment]
Yeah sorry there's currently no 1.17 version since demand was low and more versions means more work to maintain
But I'm working on some stuff that may make it easier so I'll keep this in mind
